librz/include/rz_analysis.h

```diff
@@ -2145,6 +2145,8 @@ RZ_API RzAnalysisClassErr rz_analysis_class_vtable_delete(RzAnalysis *analysis,
 RZ_API RzGraph /*<RzGraphNodeInfo *>*/ *rz_analysis_class_get_inheritance_graph(RzAnalysis *analysis);
 
 RZ_API RZ_OWN RzPVector /*<RzAnalysisVar *>*/ *rz_analysis_function_args(RzAnalysis *a, RzAnalysisFunction *fcn);
+RZ_API RZ_OWN RzPVector /*<RzAnalysisVar *>*/ *rz_analysis_function_vars(RzAnalysis *a, RzAnalysisFunction *fcn);
+RZ_API RZ_BORROW RzAnalysisVar *rz_analysis_function_get_arg_idx(RzAnalysis *analysis, RzAnalysisFunction *f, size_t index);
```

Comment:
is missing all the RZ_NONNULL/NULLABLE etc..

Reply:
Good catch, forgot about it.